en/docs/reference/pricing.md

```diff
@@ -2,229 +2,145 @@
 
-#### Update maintenance status 
 
-![Update maintenance status](../assets/img/references/webhook-example1.png)
+#### Integration and automation
 
-Let’s see how the steps are calculated for the update maintenance process:
+Choreo supports the following integration and automation features for all the plans:
 
-- As illustrated above, the webhook registers with the Google sheet at the time the Webhook component is created on Choreo.
-- The supervisor initiates the update process by updating or adding the relevant information on the [Aircraft Maintenance Sheet](#AircraftMaintenanceSheet).
-- The Google Sheet triggers **1 event to the Webhook Listener,[Aircraft Status Notifier](#aircraftNotifWebhook)** on Choreo. This is counted as **1 step**.
-- The Webhook Listener,[Aircraft Status Notifier](#aircraftNotifWebhook) on Choreo then makes **1 database call to the [Flight Scheduling Database](#flightSchedulingDatabase)** to update the flight maintenance status. This is counted as 1 step. 
+- Triggers
+- Scheduled jobs
+- 500+ out-of-the-box connectors for integration and automation
 
+#### AI-assisted development
 
-- **Assuming the entire execution time for the update process takes less than 500ms**, the update process thereby results in a **total of 2 steps**.
+Choreo supports the following AI-assisted development features for all the plans:
 
-!!! note
-    The above cost calculation is dependent on the above architecture and design of the Webhooks.
-<br/>
+- Performance predictions
+- Intelligent data mapping
+- Smart code assistance
+- Anomaly detection
 
-### Example 4: Scheduled Job 
-An airline revises its ticket prices based on the price of jet fuel. This typically happens once a day. The airline uses a scheduled task on Choreo to run this job and to update the jet fuel price once a day.
+#### API management
 
-#### Scenario 
- 
-Airlines compute ticket prices typically once per day. The ticket prices depend on a number of factors. The price of oil is one such major factor that affects the pricing.  
+Choreo supports the following API management features for all the plans:
 
-The [scheduler](#scheduler) on Choreo runs once per day and fetches the oil prices for the day from the [Oil Price API](#oilPriceAPI). It then computes the jet fuel price applying the price calculation formula. The scheduler then updates the daily jet fuel price in the [Air Fare Database](#airFareDatabase).
+| Free           |  Pay As You Go  | Enterprise |
+|----------------|-----------------|------------|
+| Creating new APIs<br/><br/>Securing APIs<br/><br/>Testing APIs<br/><br/>API Developer Portal<br/><br/>API Analytics and Insights<br/>(with only 7 days of data retention) | Creating new APIs<br/><br/>Securing APIs<br/><br/>Testing APIs<br/><br/>API Developer Portal<br/><br/>API Analytics and Insights | Creating new APIs<br/><br/>Securing APIs<br/><br/>Testing APIs<br/><br/>API Developer Portal<br/><br/>API Analytics and Insights |
```

Comment:
How many days data retention for other plans?

Reply:
Updated.